ReactQt/runtime/src/sourcecode.cpp

```diff
@@ -86,24 +88,38 @@ void SourceCode::setRetryCount(int retryCount) {
 void SourceCode::loadSource(QNetworkAccessManager* nam) {
     Q_D(SourceCode);
 
-    QNetworkRequest request(d->scriptUrl);
-    QNetworkReply* reply = nam->get(request);
-    QObject::connect(reply, &QNetworkReply::finished, [=]() {
-        reply->deleteLater();
-        if (reply->error() != QNetworkReply::NoError) {
-            if (d->retryAttempts < d->retryCount) {
-                d->retryAttempts++;
-                d->retryTimeout *= 2;
-                QTimer::singleShot(d->retryTimeout, [=] { loadSource(nam); });
-            } else {
-                qCritical() << __PRETTY_FUNCTION__ << ": Error while loading source" << reply->errorString();
-                Q_EMIT loadFailed();
+    const QString JS_BUNDLE_RESOURCES_PATH = QStringLiteral(":/index.desktop.bundle");
+    if (QFile::exists(JS_BUNDLE_RESOURCES_PATH)) {
+        QTimer::singleShot(1000, [=]() {
```

Comment:
@vkjr here happens loading of embedded JS bundle from binary's resources. Delay is a workaround to make sure that node server is started fully (creation of sandbox and precreation of Realm instances requires some time). Proper solution instead of delay, seems, might be receiving of some special message from node server, that all setup is done and we can load custom app JS code for execution.